src/librustc_session/session.rs

```diff
@@ -1140,6 +1168,16 @@ fn validate_commandline_args_with_session_available(sess: &Session) {
         }
     }
 
+    // Unwind tables cannot be disabled if the target requires them.
+    if let Some(include_uwtables) = sess.opts.cg.force_unwind_tables {
+        if sess.target.target.options.requires_uwtable && !include_uwtables {
+            sess.err(
+                "Target requires unwind tables, they cannot be disabled with \
```

Comment:
```suggestion
                "target requires unwind tables, they cannot be disabled with \
```

(to match other error messages)

Also, do we want a test for this?

Reply:
I'm not sure how to write a test that works in a way that it can be run (and pass) on multiple targets.

Reply:
Tests can be restricted to specific targets, in this case I think a compile-fail test with `// only-x86_64-windows-msvc` header should suffice.

Reply:
Let's hope I got this line right!